hive-metastore/src/main/java/org/apache/iceberg/hive/HiveCatalog.java

```diff
@@ -69,13 +73,24 @@ public HiveCatalog(Configuration conf) {
     this.conf = conf;
     this.createStack = Thread.currentThread().getStackTrace();
     this.closed = false;
+    this.fileIO = new HadoopFileIO(conf);
   }
 
   public HiveCatalog(String name, String uri, int clientPoolSize, Configuration conf) {
     this(name, uri, null, clientPoolSize, conf);
   }
 
   public HiveCatalog(String name, String uri, String warehouse, int clientPoolSize, Configuration conf) {
+    this(name, uri, warehouse, clientPoolSize, conf, Maps.newHashMap());
+  }
+
+  public HiveCatalog(
+      String name,
+      String uri,
+      String warehouse,
+      int clientPoolSize,
+      Configuration conf,
+      Map<String, String> properties) {
```

Comment:
Nit: we prefer args on as few lines as possible, rather than this style.

Reply:
I see, sorry I did not see the comments and get a chance to change it before the merge. I guess this kind of issue will be less frequent as I get more familiar with the code style here, but feel free to ping me at any time and I can fix all the style issues before merge 😃

Reply:
No problem! I made this comment just before I merged it. Commits don't need to be perfect. I just wanted to let you know for next time.